department-of-veterans-affairs/vets-website
# Update income sources receiver

## Summary
Update the conditional page for income sources on the Pension Benefits form (21p-527EZ)

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Sign in via mocked authentication at `http://localhost:3001/sign-in/mocked-auth`
3. Copy maximal-data.json into 'Save in progress' menu and return to /receives-income page
4. Select 'Yes' and continue to the 'Gross monthly income' page
5. Verify that the form fields have been updated and meets AC
6. Verify that all form data is saved to the api and redux store

## Related issue(s)
[Issue in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73161)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

## Screenshots
| Before | After | Invalid | Valid |
| ------ | ------ | ------ | ------ |
|![localhost_3001_pension_application_527EZ_financial_income-sources (3)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/20744cd4-280f-4512-9c22-5c5812b7a90a)|![localhost_3001_pension_application_527EZ_financial_income-sources (1)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/c65636b9-c07b-4464-809c-e13e239ebb16)|![localhost_3001_pension_application_527EZ_financial_income-sources (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/162260e8-3736-450b-8cc1-37c701d6e011)|![localhost_3001_pension_application_527EZ_financial_income-sources](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/73872299-3571-49d8-8da9-af00bb9f9756)



## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Issue in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73161)

### Quality Assurance & Testing

### Error Handling


### Authentication
